src/sentry/middleware/locale.py

```diff
@@ -22,17 +22,23 @@ def process_request(self, request):
         if settings.MAINTENANCE:
             return
 
+        # We don't care about locale for API
         if '/api/0/' in request.path:
             return
 
-        safe_execute(self.load_user_conf, request)
-
-        super(SentryLocaleMiddleware, self).process_request(request)
+        # Or our store endpoint
+        if request.path[-7:] == '/store/':
```

Comment:
I'm intentionally doing the slice here since it's significantly faster than `endswith()` and this is a middleware hit for every request.